src/stream/src/executor/backfill/cdc/mod.rs

```diff
@@ -17,4 +17,35 @@ mod state;
 mod upstream_table;
 
 pub use cdc_backfill::CdcBackfillExecutor;
+use risingwave_pb::stream_plan::StreamCdcScanOptions;
 pub use upstream_table::external::ExternalStorageTable;
+
+#[derive(Debug, Clone)]
+pub struct CdcScanOptions {
```

Comment:
It appears to be a duplicate of the one defined in the `frontend` crate with the same name. Can we merge them?